sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/namedExpressions.scala

```diff
@@ -255,6 +255,11 @@ case class Alias(child: Expression, name: String)(
     s"${child.sql} AS $qualifierPrefix${quoteIfNeeded(name)}"
   }
 
+  // Copying this alias with a new child expression.
+  def withChild(newChild: Expression): Alias = {
```

Comment:
```suggestion
  def withNewChild(newChild: Expression): Alias = {
```
to match the existing naming style